<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicImportDialog</class>
 <widget class="QDialog" name="BasicImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="fileLabel">
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="filePathLineEdit">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fileSelectionPushButton">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="aircraftLabel">
       <property name="text">
        <string>Aircraft:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="AircraftSelectionComboBox" name="aircraftSelectionComboBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="addToFlightCheckBox">
       <property name="text">
        <string>Add to existing flight</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="optionGroupBox">
     <property name="title">
      <string>Options</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>486</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="defaultButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AircraftSelectionComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">AircraftSelectionComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>defaultButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>BasicImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>BasicImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
